spring-cloud-task-core/src/main/resources/org/springframework/cloud/task/schema-hsqldb.sql

```diff
@@ -11,8 +12,12 @@ CREATE TABLE TASK_EXECUTION  (
 );
 
 CREATE TABLE TASK_EXECUTION_PARAMS  (
-	TASK_EXECUTION_ID VARCHAR(100) NOT NULL ,
+	TASK_EXECUTION_ID BIGINT NOT NULL ,
 	TASK_PARAM VARCHAR(250) ,
 	constraint TASK_EXEC_PARAMS_FK foreign key (TASK_EXECUTION_ID)
 	references TASK_EXECUTION(TASK_EXECUTION_ID)
 ) ;
+
+CREATE TABLE TASK_SEQ (
+	ID BIGINT IDENTITY
```

Comment:
If I run the population script once...run a few tasks...then run the script again, won't you get two rows?